client/src/components/Intro/learn-alert.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Alert, Button } from '@freecodecamp/react-bootstrap';
+import { useFeature } from '@growthbook/growthbook-react';
+import { useTranslation } from 'react-i18next';
+import { Link } from '../helpers';
+
+interface LearnAlertProps {
+  onDonationAlertClick: () => void;
+  isDonating: boolean;
+}
+
+const LearnAlert = ({
+  onDonationAlertClick,
+  isDonating
+}: LearnAlertProps): JSX.Element | null => {
+  const { t } = useTranslation();
+  const researchRecruitment = useFeature('show-research-recruitment-alert');
+  const universityCreation = useFeature('university-creation-alert');
+  const seasonalMessage = useFeature('seasonal-alert');
+
+  const researchRecruitmentAlert = (
+    <Alert>
+      <p>
+        <b>Launching Oct 19</b>: freeCodeCamp is teaming up with researchers
+        from Stanford and UPenn to study how to help people build strong coding
+        habits.
+      </p>
+      <p style={{ marginBottom: 20, marginTop: 14 }}>
+        Would you like to get involved? You’ll get free coaching from our
+        scientists.
+      </p>
+      <div style={{ display: 'flex', justifyContent: 'center' }}>
+        <Button
+          href={'https://wharton.qualtrics.com/jfe/form/SV_57rJfXROkQDDU2y'}
+        >
+          Learn about HabitLab
+        </Button>
```

Comment:
I do not like being taken away from the main site. Can we make this an `a[target="_blank"]`?